plugins/javadoc/src/main/kotlin/javadoc/JavadocPlugin.kt

```diff
@@ -3,14 +3,18 @@ package org.jetbrains.dokka.javadoc
 import javadoc.JavadocDocumentableToPageTranslator
 import javadoc.location.JavadocLocationProviderFactory
 import javadoc.renderer.KorteJavadocRenderer
+import javadoc.signatures.JavadocSignatureProvider
 import org.jetbrains.dokka.CoreExtensions
 import org.jetbrains.dokka.base.DokkaBase
+import org.jetbrains.dokka.base.resolvers.external.JavadocExternalLocationProviderFactory
+import org.jetbrains.dokka.kotlinAsJava.signatures.JavaSignatureProvider
 import org.jetbrains.dokka.plugability.DokkaPlugin
 import org.jetbrains.dokka.plugability.querySingle
 
 class JavadocPlugin : DokkaPlugin() {
 
     val dokkaBasePlugin by lazy { plugin<DokkaBase>() }
+    val javadocFormat = "javadoc"
```

Comment:
I would personally go with making it private companion object val, as this is an important static value, but thats just me

Reply:
makes sense :)